airbyte-cron/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG JDK_VERSION=19-slim-bullseye
 ARG JDK_IMAGE=openjdk:${JDK_VERSION}
 FROM ${JDK_IMAGE} AS cron
 
-ARG VERSION=0.40.0-alpha
+ARG VERSION=0.40.3
```

Comment:
Just curious if we know why this didn't get bumped by @evantahler's PR.  Do we need to add `airbyte-cron` to a script somewhere to ensure it gets updated?

Reply:
@gosusnp found the problem! `.bumpversion.cfg` is what the publish command uses to know which files to bump.  This wasn't in there.

Reply:
Yep, that should be the issue.